MTTabController.m

```diff
@@ -40,4 +40,10 @@ - (BOOL) MouseTerm_resignFirstResponder
     return YES;
 }
 
+- (void) MouseTerm_tabControllerDealloc
+{
+    [[(TTTabController *)self profile] release];
```

Comment:
Yeah I can reproduce it, I submitted this pull request https://github.com/saitoha/mouseterm-plus/pull/8 that should fix it. It's actually a more serious bug since it is trigger by all the setScripting methods. Please try and let me know!